resources/model/OpenStudio.idd

```diff
@@ -30404,6 +30404,151 @@ OS:Generator:FuelSupply,
+      \type real
+      \minimum> 0.0
+      \units m
+  N4, \field Number of Blades
+      \minimum 2
+      \required-field
+  N5, \field Rated Power
+      \required-field
+      \note This field is the nominal power at the rated wind speed.
+      \note Users should input maximum power in case of Fixed Speed Fixed Pitch control type.
+      \type real
+      \minimum> 0.0
+      \units W
+  N6, \field Rated Wind Speed
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units m/s
+  N7, \field Cut In Wind Speed
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units m/s
+  N8, \field Cut Out Wind Speed
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units m/s
+  N9, \field Fraction system Efficiency
+      \type real
+      \minimum> 0.0
+      \maximum 1.0
+      \required-field
+  N10,\field Maximum Tip Speed Ratio
+      \type real
+      \minimum> 0.0
+      \maximum 12.0
+      \required-field
+  N11,\field Maximum Power Coefficient
+      \note This field should be input if the rotor type is Horizontal Axis Wind Turbine
+      \type real
+      \minimum> 0.0
+      \maximum 0.59
+      \required-field
+  N12,\field Annual Local Average Wind Speed
+      \type real
+      \units m/s
+      \minimum> 0.0
+  N13,\field Height for Local Average Wind Speed
+      \type real
+      \units m
+      \minimum> 0.0
+      \required-field
+  N14,\field Blade Chord Area
+      \type real
+      \units m2
+  N15,\field Blade Drag Coefficient
+      \note This field is only for Vertical Axis Wind Turbine..
+      \note The user must input this field if the rotor type is Vertical Axis Wind Turbine.
+      \type real
+      \required-field
+  N16,\field Blade Lift Coefficient
+      \note This field is only for Vertical Axis Wind Turbine..
+      \note The user must input this field if the rotor type is Vertical Axis Wind Turbine.
+      \type real
+      \required-field
+  N17,\field Power Coefficient C1
+      \note This field is only available for Horizontal Axis Wind Turbine.
+      \note The user should input all six parameters
+      \note so that the analytic approximation is assumed.
+      \note The simple approximation will be assumed,
+      \note if any field C1 through C6 is not input.
+      \note Leave this field blank, if the manufacturer's data is unavailable
+      \note so that the simple approximation will be assumed.
+      \minimum> 0.0
+      \type real
+      \required-field
+  N18,\field Power Coefficient C2
+      \minimum> 0.0
+      \type real
+      \required-field
+  N19,\field Power Coefficient C3
+      \minimum> 0.0
+      \type real
+      \required-field
+  N20,\field Power Coefficient C4
+      \minimum 0.0
+      \type real
+      \required-field
+  N21,\field Power Coefficient C5
+      \minimum> 0.0
+      \type real
+      \required-field
+  N22;\field Power Coefficient C6
+      \minimum> 0.0
+      \type real
+      \required-field
```

Comment:
Make extensible and use max fields to limit to 6? @joseph-robertson 

This isn't mandatory in this case, as it's not like there will be new coefficients added.

Reply:
I went to look for the equation in the source code, and it seems I discovered a bug... During the refactor to C++ it seems that one term of the equation corresponding to C3 has been omitted: https://github.com/NREL/EnergyPlus/blob/d6cf92a2a7dc8ca3489d39792c4315f6f3bea957/src/EnergyPlus/WindTurbine.cc#L905-L907

Reply:
Well perhaps that's done on purpose,b ut both `C3` and `C4` are completely unused... I opened https://github.com/NREL/EnergyPlus/issues/8490

Reply:
Decided not to go the extensible route.